main_comment.go

```diff
@@ -131,6 +131,18 @@ func processGitHubComment(
 		if err != nil {
 			log.Error(err)
 		}
+	case strings.Contains(commentBody, commandMarkConventionalCommit) &&
+		strings.Contains(pr.GetUser().GetLogin(), dependabotName):
+		log.Infof(
+			"Attempting to amend to satisfy conventional commits in PR: %s/%d and head: %s",
```

Comment:
I think we can come up with a better log line here :smile_cat: 

How about, `Attempting to make the PR: %s/%d and commit: %s a conventional commit of type (add type)`, or something similar?

Reply:
We have yet to extract the type when executing this line :/